Gems/ROS2/Code/Source/Camera/CameraSensorConfiguration.h

```diff
@@ -19,7 +19,8 @@ namespace ROS2
         AZ_TYPE_INFO(CameraSensorConfiguration, "{386A2640-442B-473D-BC2A-665D049D7EF5}");
         static void Reflect(AZ::ReflectContext* context);
 
-        float m_verticalFieldOfViewDeg = 90.0f; //!< Vertical field of view of camera sensor.
+        float m_horizontalFieldOfViewDeg = 86.0f; //!< Horizontal field of view of camera sensor.
```

Comment:
Why have default values changed here?

Reply:
These values are the defaults for the intel realsense

Reply:
I do not like that, it will change existing default cameras, it should have version converter if you want that. Please revert.

Reply:
Reverted